src/lib/homestore_backend/snapshot_receive_handler.cpp

```diff
@@ -107,12 +107,12 @@ int HSHomeObject::SnapshotReceiveHandler::process_blobs_snapshot_data(ResyncBlob
         shared< BlobIndexTable > index_table;
```

Comment:
please add comments to explain where the duplication comes from,  they should from resent batch only if we always start from pristine